app/services/registration_transfer_service.rb

```diff
@@ -27,18 +19,56 @@ def find_transient_registration
   def find_user(email)
```

Comment:
I'd like to see this method name changed. `find_user` infers it'll search for a user with the matching email address and return it, but in fact it'll go on and complete the transfer, then send a status result back. 
```suggestion
  def transfer_existing_user(email)
```